CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 ### Added
 
 - [#579](https://github.com/bumble-tech/appyx/pull/579) – Expose `AndroidLifecycle` in `PlatformLifecycleRegistry` for Android
+- [#601](https://github.com/bumble-tech/appyx/pull/601) – Support iOS platform for  **:appyx-navigation** & adding sample iOS targets in **:demos:appyx-navigation** & **:demos:appyx-interactions**  
```

Comment:
I'd suggest to have the same exact text here as the title of the referenced ticket. If more info is needed, it can always be added into the description of the ticket.

Reply:
Updated